src/runtime/math.py

```diff
@@ -215,25 +227,68 @@ def fabs(x: f64) -> f64:
         return -x
     return x
 
+@overload
+def fabs(x: i64) -> i64:
+    """
+    Return the absolute value of `x`.
+    """
+    if x < 0:
+        return -x
+    return x
```

Comment:
Might not be the best way to implement but fine for now.

Reply:
I just noticed the mistake, correcting it it in the next commit